.env.sample

```diff
@@ -0,0 +1,4 @@
+DB_URL = mongodb://localhost

```

Comment:
🍬 you can separate the protocol from the address as well

(i'll use the 🍬_(candy emoji)_ if something is just an opinion)